src/registrar/models/domain.py

```diff
@@ -1401,6 +1401,8 @@ def _disclose_fields(self, contact: PublicContact):
         DF = epp.DiscloseField
```

Comment:
Line 1399, change 'visable' to 'visible'

Reply:
Good catch! Just updated the spelling